libtelemetry/src/main/java/com/mapbox/android/telemetry/CertificatePinnerFactory.java

```diff
@@ -48,7 +48,7 @@ private void addCertificatesPins(Map<String, List<String>> pins, CertificatePinn
 
   private Map<String, List<String>> removeBlacklistedPins(Map<String, List<String>> pins,
                                                           CertificateBlacklist certificateBlacklist) {
-    List<String> blackList = certificateBlacklist.retrieveBlackList();
+    List<String> blackList = certificateBlacklist.getRevokedKeys();
```

Comment:
this line is still called with every request, do we actually need to do this? should this be only called every 24 hours when blacklist is potentially updated?